src/azure-cli-core/azure/cli/core/tests/test_profile.py

```diff
@@ -597,7 +622,8 @@ def test_find_subscriptions_in_vm_with_msi_user_assigned_with_res_id(self, creat
     @mock.patch('azure.cli.core.auth.identity.Identity.get_user_credential', autospec=True)
     @mock.patch('azure.cli.core.auth.identity.Identity.login_with_auth_code', autospec=True)
     @mock.patch('azure.cli.core._profile.can_launch_browser', autospec=True, return_value=True)
-    def test_login_no_subscription(self, can_launch_browser_mock,
+    @mock.patch('azure.cli.core._profile.is_github_codespaces', autospec=True, return_value=False)
+    def test_login_no_subscription(self, is_github_codespaces_mock, can_launch_browser_mock,
```

Comment:
We don't support running these tests in GitHub Codespaces, so I don't think it is necessary to patch `is_github_codespaces` in these test methods.